### PR TITLE
docs: adding restAddress configuration for integration with Zeebe API

### PR DESCRIPTION
**Pull request by marcosgvieira**
## Description

I'm adding an extra property that can be used to configure communication between Tasklist and Zeebe Rest API.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by houssain-barouni — DISMISSED** [review]

**Comment by akeller** [comment]
@marcosgvieira is this technically part of the 8.5 release? It should also be included in the `/docs` directory so it can be available in the 8.6 preview.

I'm also asking if we need to review and merge quickly so we can release this change ASAP.

File: versioned_docs/version-8.5/self-managed/tasklist-deployment/tasklist-configuration.md [hunk 1]
```diff
@@ -126,11 +126,12 @@ Tasklist needs a connection to Zeebe broker to start the import.
...
+| Name                                   | Description                                                                                                         | Default value         |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------- | --------------------- |
+| camunda.tasklist.zeebe.gatewayAddress  | Gateway address that points to Zeebe as hostname and port.                                                          | localhost:26500       |
+| camunda.tasklist.zeebe.secure          | Connection should be secure via Transport Layer Security (TLS).                                                     | false                 |
+| camunda.tasklist.zeebe.certificatePath | Path to certificate used by Zeebe. This is necessary when the certificate isn't registered in the operating system. | -                     |
+| camunda.tasklist.zeebe.restAddress     | Path to Zeebe rest address. This is necessary to consume Zeebe API from Tasklist                                    | http://localhost:8083 |
```
[the remark this answers is not in the document]
Comment (marcosgvieira) [reply versioned_docs/version-8.5/self-managed/tasklist-deployment/tasklist-configuration.md:134]: @houssain-barouni yes, we delivered 8.5 with this default, after we changed to improve e2e-tests, we need another PR for 8.5.1.

**Comment by marcosgvieira** [comment]
> @marcosgvieira is this technically part of the 8.5 release? It should also be included in the `/docs` directory so it can be available in the 8.6 preview.
> 
> I'm also asking if we need to review and merge quickly so we can release this change ASAP.

@akeller I missed the commit of the file in the /docs, I just added it. Yes, it's part of 8.5, the customers initially would not touch this property but is never bad to have it documented.

File: docs/self-managed/tasklist-deployment/tasklist-configuration.md [hunk 2]
```diff
@@ -126,11 +126,12 @@ Tasklist needs a connection to Zeebe broker to start the import.
...
+| Name                                   | Description                                                                                                         | Default value         |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------- | --------------------- |
+| camunda.tasklist.zeebe.gatewayAddress  | Gateway address that points to Zeebe as hostname and port.                                                          | localhost:26500       |
+| camunda.tasklist.zeebe.secure          | Connection should be secure via Transport Layer Security (TLS).                                                     | false                 |
+| camunda.tasklist.zeebe.certificatePath | Path to certificate used by Zeebe. This is necessary when the certificate isn't registered in the operating system. | -                     |
+| camunda.tasklist.zeebe.restAddress     | Path to Zeebe rest address. This is necessary to consume Zeebe API from Tasklist                                    | http://localhost:8083 |
```
Comment (akeller) [inline docs/self-managed/tasklist-deployment/tasklist-configuration.md:134]:
```suggestion
| camunda.tasklist.zeebe.restAddress     | Path to Zeebe REST address. This is necessary to consume the Zeebe API from Tasklist.                                    | http://localhost:8083 |
```

File: versioned_docs/version-8.5/self-managed/tasklist-deployment/tasklist-configuration.md (hunk 1 above)
Comment (akeller) [inline versioned_docs/version-8.5/self-managed/tasklist-deployment/tasklist-configuration.md:134]:
```suggestion
| camunda.tasklist.zeebe.restAddress     | Path to Zeebe REST address. This is necessary to consume the Zeebe API from Tasklist.                                    | http://localhost:8083 |
```

**Comment by akeller** [comment]
I'll clean this up so we can get this merged 😎

**Review by akeller — APPROVED** [review]

**Pull request merged**